ntbs-service/Pages/Notifications/NotificationModelBase.cs

```diff
@@ -34,7 +35,7 @@ public abstract class NotificationModelBase : PageModel
         public IList<Alert> Alerts { get; set; }
 
         [BindProperty]

```

Comment:
While we're here - I don't think this wants to be a bind property, we never want to populate it from the request!